### DISCUSSION TITLE
We need an official way to hide title bars!

### DISCUSSION THREAD

**Discussion by igorsantos07** [linuxmint/discussions#644] [Ideas]
This feature is implemented in a long-abandoned extension called [Maximus](https://cinnamon-spices.linuxmint.com/extensions/view/73), which so far is the 4th most voted on the store. A [sibling extension that moved the title bar to the top panel](https://github.com/fatihmete/window-buttons-with-title) got removed for possibly lack of interest from developers, but [not without a couple of people trying to revive it](https://github.com/linuxmint/cinnamon-spices-applets/issues/1877).

There's also quite some interest in forums about this: [[1]](https://forums.linuxmint.com/viewtopic.php?t=417244) [[2]](https://www.reddit.com/r/linuxmint/comments/h0vlyd/how_to_remove_the_title_bar/) [[3]](https://forums.linuxmint.com/viewtopic.php?t=257382)

This feature is very useful for keyboard-centric users, which usually don't need the mouse to work their windows around.

Is there _any_ chance whatsoever that Maximus could be officially adopted by Cinnamon developers and migrated to become part of official source, or maybe even become an option in the Themes settings screen (under a severe warning, obviously)?

Please please?? Maximus is mostly working fine so far, but like its sibling extension, it's under severe risk of going under with future Cinnamon changes.

**Comment by JosephMcc** [comment]
There is pretty much zero chance of this being added to Cinnamon.
